Tools/Scripts/generate-dirty-files

```diff
@@ -179,6 +199,32 @@ def find_all_results(args):
         results_msg += f'{proj} ({count}) '
     print(results_msg)
 
+    return results_for_upload
+
+
+def upload_results(options, results):
+    config_options = eval(options.configuration)
+    config = Upload.create_configuration(
+        architecture=config_options.get('architecture') or None,
+        platform=config_options.get('platform') or None,
+        version=config_options.get('version') or None,
+        version_name=config_options.get('version_name') or None,
+        style=config_options.get('style') or None,
+        sdk=config_options.get('sdk') or None,
+    )
```

Comment:
Especially given we already rely on `webkitpy`, maybe we should just use `Port.configuration_for_upload` again?

Reply:
`Port.configuration_for_upload` relies on having a port, which these tests don't have. I'm open to solutions that don't rely on passing in configs as an option but I don't know if there are any good ones.